PubNub/Data/Service Objects/PNSubscriberResults.h

```diff
@@ -27,6 +27,8 @@
  */
 @property (nonatomic, readonly, strong) NSNumber *timetoken;
 
+@property (nonatomic, readonly, strong) NSNumber *region;
```

Comment:
@jzucker2 Same questions as in notes for changes in _PNSubscribeStatus.h_ about whether it is advisability to expose this information to the user (and suggestion how we can do it if it will be required).